### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by El-Murtazina**
Elina Murtazina el.murtazina@innopolis.university

### CODE REVIEW

File: lib/templates/lab1/lab1.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import 'dart:async';
+
 import 'package:flutter/material.dart';
```
Comment (timur-harin) [inline lib/templates/lab1/lab1.dart:4]: Why is it here?

File: lib/templates/lab7/album.dart [hunk 2]
```diff
@@ -3,10 +3,13 @@ import 'package:freezed_annotation/freezed_annotation.dart';
...
-// TODO code generate class with needed parameters from
-// https://jsonplaceholder.typicode.com/albums/1
 @freezed
 class Album with _$Album {
-  // TODO add fromJson factory
-  
+  const factory Album({
+    required int userId,
+    required int id,
+    required String title,
```
Comment (timur-harin) [inline lib/templates/lab7/album.dart:11]: Why is it here?

File: lib/templates/middleAssignment/main.dart [hunk 3]
```diff
@@ -14,8 +23,105 @@ class MiddleAssigmentApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.blue,
```
Comment (timur-harin) [inline lib/templates/middleAssignment/main.dart:24]: Why is it here?

File: test/unit_test.dart [hunk 4]
```diff
@@ -11,26 +9,26 @@ void main() {
...
   test('mod two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.mod(7, 3), 1);
   });
 
   test('power two numbers', () {
     final calculator = Calculator();
-    // TODO add expect
+    expect(calculator.power(2, 3), 8);
```
Comment (timur-harin) [inline test/unit_test.dart:32]: You did not added it in Calculator class

Comment (El-Murtazina) [reply test/unit_test.dart:32]: sorry, what did i not add?  Power function is in calculator.dart

Comment (timur-harin) [reply test/unit_test.dart:32]: you are right

Comment (timur-harin) [reply test/unit_test.dart:32]: missed it

Comment (timur-harin) [reply test/unit_test.dart:32]: Updated +1 point

**Pull request closed without merging**